tethne/readers/dfr.py

```diff
@@ -214,7 +214,7 @@ def read(path, corpus=True, index_by='doi', **kwargs):
        >>> from tethne.readers import dfr
        >>> papers = dfr.read("/Path/to/DfR")
     """
-    parser = DfRParser(os.path.join(path, "citations.XML"))
+    parser = DfRParser(os.path.join(path, "citations.xml"))
```

Comment:
Older versions of DfR had files called ``citations.XML``. One of the test datasets has a file by that name...